src/getting_started/changelogs/Changelogs.js

```diff
@@ -14,6 +14,90 @@ export default function Authentication() {
         API V4 since 2017-09-18.  Please check here regularly for updates.
         </p>
       </section>
+      <section>
+        <h2>2018-02-21</h2>
+        <hr /><br />
+        <b>Breaking:</b><br />
+        <ul>
+          <li>Unified IPv4 and IPv6 Objects
+            <ul>
+              <li>Removed "range" from IPv6 object</li>
+              <li>Added "linode_id" to IPv6 object</li>
+              <li>Added "region" to IPv6 object</li>
+            </ul>
+          </li>
+          <li>Separated /networking/ipv6 into /pools and /ranges
+            <ul>
+              <li>Added /networking/ipv6/pools</li>
```

Comment:
Should this say `Added GET /networking/ipv6/pools`?

If you are specifying GET in the /networking/ipv6 bullet below, should we be consistent?